.github/workflows/versions.yml

```diff
@@ -0,0 +1,116 @@
+name: Update or Verify Versions
+
+# Currently this workflow relies on one secret:
+# - GITHUB_TOKEN: Required for creating a release on GitHub
+# It can be passed using the `secrets: inherit` option in the caller workflow file.
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'New version to use for the extension. Example: 1.5.2'
+        required: true
+        type: string
+
+      branch:
+        description: 'Branch to be used when updating versions'
+        required: true
+        type: string
+
+      dependencies:
+        description: |
+          A comma-separated list of dependencies with their versions. 
+          Each dependency can optionally specify the file paths where it applies using the `@` symbol.
+          
+          **Syntax:** `<name> <version>[@file_path1[,file_path2,...]]`
+
+          - If the `@` syntax is used, the dependency will only apply to the specified files.
+          - When specifying custom files, you must provide either an absolute or relative path to each file.
+          - If the `@` symbol is omitted, the dependency applies to all relevant files.
+
+          **Example:**
+          `"AEPCore 3.1.1, AEPServices 8.9.10@AEPCore.podspec, Edge 3.2.1@Package.swift"`
```

Comment:
Update this so that only the dependency version is specified during the workflow run, while other details like file names are hard coded as environment variables in the config file.